lib/index.js

```diff
@@ -110,6 +110,10 @@ var flatten = function(options) {
         });
     }
 
+    if (json.path && typeof json.path === 'string') {
```

Comment:
Would `json.path` ever not be a string?

Reply:
It should never be anything but a string. I did this for extra safety, but can remove it.